Frameworks/include/UIViewInternal.h

```diff
@@ -33,6 +33,27 @@
 @class WXFrameworkElement;
 @class WUXIPointerRoutedEventArgs;
 
+// Round subpixel values to be able to perform per-pixel UI placement/calculations
+inline float doPixelRound(float f) {
```

Comment:
I may have seen it on images as well - not sure but definitively on label.

---
In reply to: [100638812](https://github.com/Microsoft/WinObjC/pull/1920#discussion_r100638812) [](ancestors = 100638812,100634412)